### ISSUE TITLE
non-Pull Request workflows causing warning and no plan is posted to the job summary

### ISSUE THREAD

**Issue by leuthelt** [liatrio/terraform-change-pr-commenter#61]
Hello,

I'm using below **GitHub Workflow configuration**:
```
- name: Comment Terraform Plan results in PR and as job summary
  uses: liatrio/terraform-change-pr-commenter@v1.4.5
  with:
    json-file: ${{ env.WORKING_DIR }}/tfplan.json
    expand-comment: 'true'
    include-plan-job-summary: 'true'
```

**Actual behaviour:**
During a pull request everything works fine. Comments are posted to PR and job summary as well. 
But when I'm running workflow in a non-pull request, I'm receiving following warnings:
```
Warning: Action doesn't seem to be running in a PR workflow context.
Warning: Skipping comment creation.
```

**Expected behaviour:**
Because README says _"When include-plan-job-summary = true, if the action is executed in non-Pull Request workflows, **the plan output will also be posted to the job summary of that run**. If you do not wish to have this behavior, apply conditional logic to your workflow file."_, I expect posting results to job summary during non-pull request.

Thanks for your help,
Thomas

**Comment by RobertKelly** [comment]
Thanks. We will validate and get this updated.

**Comment by jburns24** [comment]
Validated this is a bug in v1.5.0 using the steps posted in this issue. Fix incoming

**Comment by jburns24** [comment]
@leuthelt This should be fixed in [v1.6.0](https://github.com/liatrio/terraform-change-pr-commenter/releases/tag/v1.6.0)
